### ISSUE TITLE
Layout: fixed footer

### ISSUE THREAD

**Issue by nnixaa** [akveo/nebular#237]
### Issue type

**I'm submitting a ...**  (check one with "x")

* [x] bug report
* [ ] feature request

### Issue description

**Current behavior:**

Looks like `fixed` attribute doesn't change footer behavior
**Expected behavior:**

Remove or fix `fixed` footer attribute
**Steps to reproduce:**

**Related code:**
<!-- 
If you are able to illustrate the bug or feature request with an example, please provide a sample application via one of the following means:
https://github.com/akveo/ngx-admin/issues/1389

**Comment by EatonZ** [comment]
I've noticed `fixed` doesn't seem to change the footer behavior as of 6/29/18 - is this the reason why?
At this point is it safe to say it will probably be removed?

**Comment by ruisebastiao** [comment]
I'm waiting for the fix, i would like to have the footer fixed

**Comment by ProDInfo** [comment]
Please correct this behavior. Is usefull for mobile applications
Only add to css this code to src/framework/theme/components/layout/layout.component.scss:

  ::ng-deep nb-layout-footer  {
   ...
   ...

    &.fixed {
      position: fixed;
      bottom: 0;
      left: 0;
      right: 0;
      z-index: 1040;
    }
  }

**Comment by RohanGautam** [comment]
still an issue!

**Comment by kingjordan** [comment]
still an issue for me as well, however the issue is only on safari, all other browsers work fine.  THe demo on [online demo](https://www.akveo.com/ngx-admin/pages/dashboard?utm_campaign=ngx_admin%20-%20demo%20-%20ngx_admin%20docs&utm_source=ngx_admin&utm_medium=referral&utm_content=landing_main_section) works, but once I load my own angular project, the footer does not get fixed in safari browsers

**Comment by RohanGautam** [comment]
@kingjordan interesting, I'm using Firefox on Linux and it doesn't work for me there either.
@ProDInfo Your solution might work! Could you open a pull request?

**Comment by RezaRahmati** [comment]
although it's mentioned in the docs, but actually doesn't work

> NbLayoutFooterComponent
> Page footer. Located under the nb-layout content (specifically, under the columns). Could be made fixed, becoming sticky to the bottom of the view port (window).

**Comment by NicmeisteR** [comment]
This seems to still be an issue, I've submitted a merge request with a fix.

**Comment by CourserStudio** [comment]
Any workaround ? The `fixed` attribute is not work.

**Comment by NicmeisteR** [comment]
@CourserStudio nope, still no feedback on my merge on a fix and it's been over a year.
https://github.com/akveo/nebular/pull/2694
